modules/packages/VisualDebug.chpl

```diff
@@ -130,50 +132,28 @@ private proc VDebugTree (what: vis_op, name: string, time: real, id: int = 0,
   :arg tagname: name of the tag
 */
   proc tagVdebug ( tagname : string ) {
-    //coforall l in Locales[1..] do
-    //  on l do chpl_vdebug_tag (tagname.localize().c_str(), 0);
-    //chpl_vdebug_tag (tagname.localize().c_str(), 0);
     if (VisualDebugOn) {
-       VDebugTree (vis_op.v_tag, tagname, 0);
+       chpl_vdebug_tagname (tagname.c_str(), tagno);
```

Comment:
This seems really parallel-unsafe (global var updated without any protection). Don't we need something here to serialize execution when more than one task arrives here at the same time?  Or is there something else that prevents that from happening?